benchmark/framework/parser/file_access.go

```diff
@@ -0,0 +1,134 @@
+package bparser
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"sort"
+	"strings"
+	"time"
+)
+
+var (
+	outputDir      = "./output/"
+	sociLogs       = outputDir + "soci-snapshotter-stderr"
+	FileAccessLogs = outputDir + "fileAccessPatterns"
+)
+
+type FileAccessPatterns struct {
+	ContainerStartTime time.Time   `json:"containerStartTime"`
+	Operations         []Operation `json:"operations"`
+}
+
+type BaseOperation struct {
+	Level     string    `json:"level"`
+	Msg       string    `json:"msg"`
+	Operation string    `json:"operation"`
+	Path      string    `json:"path"`
+	Time      time.Time `json:"time"`
+}
+
+type Operation struct {
+	Operation                 string `json:"operation"`
+	Path                      string `json:"path"`
+	FirstAccessTimeAfterStart string `json:"firstAccessTimeAfterStart"`
+	Count                     int    `json:"count"`
+}
+
+func ParseFileAccessPatterns() error {
+
+	fa, err := getFileAccessStruct()
+	if err != nil {
+		return err
+	}
+
+	file, err := os.Open(sociLogs)
+	if err != nil {
+		return err
+	}
+
+	defer func() {
+		if err = file.Close(); err != nil {
+			return
+		}
+	}()
+	b, err := ioutil.ReadAll(file)
```

Comment:
[ioutil](https://pkg.go.dev/io/ioutil) is deprecated as of Go 1.16. Can you go with io's ReadAll?

Reply:
Sure.